launch/faulkner_sim.launch

```diff
@@ -17,7 +17,11 @@
   <arg name="scan_sector_size" default="0.628319"/> <!-- 36 degrees -->
   <arg name="avoidance_ang_vel_magnitude" default="0.3"/>
   <arg name="avoid_distance" default="0.5"/>
+  <arg name="step_x" default="5" />
+
   <arg name="dev" default="/dev/input/js0" />
+  <arg name="joyType" default="zerodelay" />  <!-- possible values xbox,zerodelay-->
+  <param name="joyType" value="$(arg joyType)"/>
```

Comment:
Create a new launch file for simulation + joystick. We'll have one launch for for simulation + keyboard and another for simulation + joystick.